gapic-generator-cloud/templates/cloud/client/_resource.erb

```diff
@@ -19,7 +19,7 @@ def <%= resource.path_helper %> <%= args.join ", " %>
 <%- resource.arguments.each do |arg| -%>
   raise ArgumentError, "<%= arg.name %> is required" if <%= arg.name %>.nil?
 <%- if arg.regexp -%>
-  raise ArgumentError, "<%= arg.msg %>" unless <%= arg.regexp %>.match <%= arg.name %>
+  raise ArgumentError, <%= arg.msg.inspect %> unless <%= arg.regexp %>.match <%= arg.name %>
```

Comment:
nit: `<%= arg.msg.inspect %>` needs to be wrapped in quotes.

Reply:
No, the `.inspect` adds the outer quotes. This is needed to ensure that characters that should be escaped are properly escaped. We use this approach in other templates.

Reply:
Thanks for the clarification!